openssh/files/sshd_config

```diff
@@ -128,8 +128,14 @@
 {{ option_default_uncommented('RSAAuthentication', 'yes') }}
 {{ option_default_uncommented('PubkeyAuthentication', 'yes') }}
 {{ option('AuthorizedKeysFile', '%h/.ssh/authorized_keys') }}
-{{ option('AuthorizedKeysCommand', 'none') }}
-{{ option('AuthorizedKeysCommandUser', 'nobody') }}
+{%- if server.authorized_keys_command is defined %}
+AuthorizedKeysCommand {{ server.authorized_keys_command.command }}
```

Comment:
I believe that server gets defined in map.jinja with "}, merge=salt['pillar.get']('openssh:server')) %}", but yes. I don't see that there is a AuthorizedKeysCommand supplied in the jinja.map. It should be commented out with the default value if unsupplied.